source/opt/ir_context.h

```diff
@@ -576,6 +576,24 @@ class IRContext {
     return GetFunction(inst->result_id());
   }
 
+  // Updates DebugInlinedAt of |inst| and analyzes use information
+  // of DebugInlinedAt.
+  void UpdateDebugInlinedAt(Instruction* inst, uint32_t dbg_inlined_at) {
```

Comment:
I don't think you need this function.  You can move the update of the debug info manager to the instruction classes version of the function.

Reply:
done